### PR TITLE
Updating naming convention to ATAT

### PR DESCRIPTION
**Pull request by danielgottlieb-ctr**
updating naming to be purely ATAT

### CODE REVIEW

**Comment by kylelaker-ccpo** [comment]
These modifications to the `alembic` scripts may actually break the migrations. I am not totally sure of that; we'll just need to pay special attention to that during the review. And we may actually need a _new_ migration that handles renaming JEDI->ATAT. Again, not sure; we just want to keep it in mind during testing and review (because the pipeline doesn't necessarily validate those),

File: templates/emails/base.txt [hunk 1]
```diff
@@ -1,9 +1,9 @@
 {% block content %}{% endblock %}
 
-What is JEDI Cloud?
-JEDI Cloud is a DoD enterprise-wide solution for commercial cloud services.
+What is ATAT?
+ATAT is a DoD enterprise-wide solution for commercial cloud services.
```
Comment (whit1206) [inline templates/emails/base.txt:4]: change to "ATAT (Account Tracking and Automation Tool) is a web-based application that streamlines the cloud onboarding process for DoD and its Mission Partners"

File: translations.yaml [hunk 2]
```diff
@@ -14,12 +14,12 @@ audit_log:
...
 base_public:
   login: Login
-  title_tag: JEDI Cloud
+  title_tag: ATAT
 home:
   add_portfolio_button_text: Add New Portfolio
   new_portfolio: New Portfolio
   get_started: Get Started
-  head: JEDI Cloud Services
+  head: ATAT Services
```
Comment (whit1206) [inline translations.yaml:22]: change to "Cloud Services"

File: translations.yaml [hunk 3]
```diff
@@ -90,14 +90,14 @@ errors:
   not_found: Page Not Found
   not_found_sub: This page does not exist.
 email:
-  application_invite: "{inviter_name} has invited you to a JEDI Cloud application"
+  application_invite: "{inviter_name} has invited you to an ATAT application"
```
Comment (whit1206) [inline translations.yaml:93]: change to "a cloud application"

File: translations.yaml [hunk 4]
```diff
@@ -90,14 +90,14 @@ errors:
...
   app_role_created:
     subject: Application Role Created
     body: "Your application role has been created.\nVisit {url}, and use your username, {username}, to login."
   azure_account_update:
     subject: Azure account update
     body: "There has been an update to your Azure account. \nVisit {url}, and use your username, {username}, to login."
-  portfolio_invite: "{inviter_name} has invited you to a JEDI Cloud portfolio"
+  portfolio_invite: "{inviter_name} has invited you to an ATAT portfolio"
```
Comment (whit1206) [inline translations.yaml:100]: change to 'a cloud portfolio"

File: translations.yaml [hunk 5]
```diff
@@ -214,7 +214,7 @@ flash:
     message: Your session expired due to inactivity. Please login again to continue.
   success: Success!
   task_order_number_error:
-    message: "The TO number has already been entered for a JEDI task order #{to_number}. Please double-check the TO number you are entering. If you believe this is in error, please contact support@cloud.mil."
+    message: "The TO number has already been entered for an ATAT task order #{to_number}. Please double-check the TO number you are entering. If you believe this is in error, please contact support@cloud.mil."
```
Comment (whit1206) [inline translations.yaml:217]: just remove "an ATAT"

File: translations.yaml [hunk 6]
```diff
@@ -367,9 +367,9 @@ login:
     learn_more: Learn more
     message: "When you are prompted to select a certificate, please select an Authentication (Identification) Certificate from the provided choices."
     title: Certificate Selection
-  h1_title: Access the JEDI Cloud
+  h1_title: Access ATAT
   login_button: Sign in with CAC
-  title_tag: Sign in | JEDI Cloud
+  title_tag: Sign in | ATAT
```
Comment (whit1206) [inline translations.yaml:372]: remove "| ATAT"

File: translations.yaml [hunk 7]
```diff
@@ -367,9 +367,9 @@ login:
     learn_more: Learn more
     message: "When you are prompted to select a certificate, please select an Authentication (Identification) Certificate from the provided choices."
     title: Certificate Selection
-  h1_title: Access the JEDI Cloud
+  h1_title: Access ATAT
```
Comment (whit1206) [inline translations.yaml:370]: change to "Access Cloud"

File: translations.yaml [hunk 8]
```diff
@@ -508,7 +508,7 @@ portfolios:
     duration:
       header: Funding duration
       tooltip: Funding duration is the period of time that there is a valid task order funding the portfolio.
-    estimate_warning: Reports displayed in JEDI are estimates and not a system of record. To manage your costs, go to Azure by selecting the Login to Azure button above.
+    estimate_warning: Reports displayed in ATAT are estimates and not a system of record. To manage your costs, go to Azure by selecting the Login to Azure button above.
```
Comment (whit1206) [inline translations.yaml:511]: change first "Azure" to "the CSP" and change "Login to Azure" to "Login"

File: uitests/framework/page_objects/__init__.py [hunk 9]
```diff
@@ -47,11 +47,11 @@ def validate_atat(self):
...
         )
 
-    # Validating JEDI Cloud Services is displayed
-    def validate_jedi(self):
+    # Validating ATAT Services is displayed
+    def validate_services(self):
         WebDriverWait(self.driver, 30).until(
             EC.text_to_be_present_in_element(
-                (By.CSS_SELECTOR, "div.home__content"), "JEDI Cloud Services"
+                (By.CSS_SELECTOR, "div.home__content"), "ATAT Services"
```
Comment (whit1206) [inline uitests/framework/page_objects/__init__.py:54]: Change all 'ATAT Services" to "Cloud Services" in all places it still exists. There is no such thing as ATAT services

File: uitests/framework/test_cases/test_new_user.py [hunk 10]
```diff
@@ -59,7 +59,7 @@ def test_new_user(self, setup):
         try:
             WebDriverWait(self.driver, 30).until(
                 EC.text_to_be_present_in_element(
-                    (By.CSS_SELECTOR, "div.home__content"), "JEDI Cloud Services"
+                    (By.CSS_SELECTOR, "div.home__content"), "ATAT Services"
```
Comment (whit1206) [inline uitests/framework/test_cases/test_new_user.py:62]: Change to Cloud Services

File: uitests/html/New_App_Step_3.html [hunk 11]
```diff
@@ -122,7 +122,7 @@
 <tr>
 <td>assertText</td>
 <td>css=.home__content > h1</td>
-<td>JEDI Cloud Services</td>
+<td>ATAT Services</td>
```
Comment (whit1206) [inline uitests/html/New_App_Step_3.html:125]: Change to "Cloud Services"

File: uitests/html/New_Portfolio.html [hunk 12]
```diff
@@ -53,7 +53,7 @@
 <tr>
 <td>assertText</td>
 <td>css=.home__content > h1</td>
-<td>JEDI Cloud Services</td>
+<td>ATAT Services</td>
```
Comment (whit1206) [inline uitests/html/New_Portfolio.html:56]: change

**Review by whit1206 — CHANGES_REQUESTED** [review]
Please change all "ATAT Sevices" to "Cloud Services" amongst other changes. Thanks

File: README.md [hunk 13]
```diff
@@ -288,8 +288,8 @@ All config settings must be declared in "config/base.ini", even if they are null
 - `CDN_ORIGIN`: URL for the origin host for asset files.
 - `CELERY_DEFAULT_QUEUE`: String specifying the name of the queue that background tasks will be added to.
 - `CELERYBEAT_SCHEDULE_VALUE`: Integer specifying a default value of how many seconds wait between scheduled celery beat tasks. All celery beat tasks use this value.
-- `CONTRACT_END_DATE`: String specifying the end date of the JEDI contract. Used for task order validation. Example: 2019-09-14
-- `CONTRACT_START_DATE`: String specifying the start date of the JEDI contract. Used for task order validation. Example: 2019-09-14.
+- `CONTRACT_END_DATE`: String specifying the end date of the Cloud contract. Used for task order validation. Example: 2019-09-14
```
[the remark this answers is not in the document]
Comment (danielgottlieb-ctr) [reply README.md:291]: done

File: templates/emails/application/invitation.txt [hunk 14]
```diff
@@ -2,8 +2,8 @@
 
 {% block content %}
 
-Join this JEDI Cloud Application
-{{ owner }} has invited you to join a JEDI Cloud Application. Login now to view or use your JEDI Cloud resources.
+Join this ATAT Application
+{{ owner }} has invited you to join a ATAT Application. Login now to view or use your ATAT resources.
```
[the remark this answers is not in the document]
Comment (danielgottlieb-ctr) [reply templates/emails/application/invitation.txt:6]: all have been updated

File: templates/emails/base.txt [hunk 15]
```diff
@@ -1,9 +1,9 @@
...
 
-What is a JEDI Cloud Portfolio?
-A JEDI Cloud Portfolio is where you may access and manage the cloud resources  associated with your applications and environments.
+What is an ATAT Portfolio?
+An ATAT Portfolio is where you may access and manage the cloud resources  associated with your applications and environments.
 
-JEDI Cloud is managed by the Cloud Computing Program Office. Learn more at jedi.cloud.
+ATAT is managed by the Cloud Computing Program Office. Learn more at atat.cloud.
```
[the remark this answers is not in the document]
Comment (danielgottlieb-ctr) [reply templates/emails/base.txt:9]: done

File: templates/emails/portfolio/invitation.txt [hunk 16]
```diff
@@ -2,8 +2,8 @@
 
 {% block content %}
 
-Join this JEDI Cloud Portfolio
-{{ owner }} has invited you to join a JEDI Cloud Portfolio.  Login now to view or use your JEDI Cloud resources.
+Join this ATAT Portfolio
+{{ owner }} has invited you to join a ATAT Portfolio.  Login now to view or use your ATAT resources.
```
[the remark this answers is not in the document]
Comment (danielgottlieb-ctr) [reply templates/emails/portfolio/invitation.txt:6]: done

File: templates/emails/base.txt (hunk 1 above)
Comment (danielgottlieb-ctr) [reply templates/emails/base.txt:4]: updated

File: translations.yaml (hunk 8 above)
Comment (danielgottlieb-ctr) [reply translations.yaml:511]: Done

File: uitests/framework/page_objects/__init__.py (hunk 9 above)
Comment (danielgottlieb-ctr) [reply uitests/framework/page_objects/__init__.py:54]: Done

File: uitests/framework/test_cases/test_new_user.py (hunk 10 above)
Comment (danielgottlieb-ctr) [reply uitests/framework/test_cases/test_new_user.py:62]: Done

File: uitests/html/New_App_Step_3.html (hunk 11 above)
Comment (danielgottlieb-ctr) [reply uitests/html/New_App_Step_3.html:125]: Done

File: uitests/html/New_Portfolio.html (hunk 12 above)
Comment (danielgottlieb-ctr) [reply uitests/html/New_Portfolio.html:56]: done

File: translations.yaml (hunk 5 above)
Comment (danielgottlieb-ctr) [reply translations.yaml:217]: done

File: translations.yaml (hunk 2 above)
Comment (danielgottlieb-ctr) [reply translations.yaml:22]: done

File: translations.yaml (hunk 3 above)
Comment (danielgottlieb-ctr) [reply translations.yaml:93]: done

File: translations.yaml (hunk 4 above)
Comment (danielgottlieb-ctr) [reply translations.yaml:100]: done

File: translations.yaml (hunk 6 above)
Comment (danielgottlieb-ctr) [reply translations.yaml:372]: done

File: translations.yaml (hunk 7 above)
Comment (danielgottlieb-ctr) [reply translations.yaml:370]: done

**Comment by kylelaker-ccpo** [comment]
Seeing that #2439 has now been opened, I am going to close this PR.

**Pull request closed without merging**